### DISCUSSION TITLE
Force ldapsearch auth?

### DISCUSSION THREAD

**Discussion by helmut72** [kanidm/kanidm#3318] [Q&A] [answered]
Output of samba 4:
```
ldapsearch -x -H ldaps://samba4.example.com:636 -b 'dc=example,dc=com' -x '+' '+'
```
... shows `Operation unavailable without authentication`.

Output of kandim (v1.4.5):
```
ldapsearch -x -H ldaps://kanidm.example.com:3636 -b 'dc=example,dc=com' -x '+' '+'`
```
... shows far too much for anonymous. Every user, admin, ...

Is there a setting to force ldapsearch auth / disable anonymous ldapsearch?

Thank you.

**Comment by Firstyear** [comment, marked as answer]
https://kanidm.github.io/kanidm/stable/accounts/anonymous_account.html

The anonymous account can be disabled (expired).

**Reply by helmut72** [reply]
Thanks, works!

What is meant with "stateless unix client"?

**Reply by Firstyear** [reply]
https://kanidm.github.io/kanidm/stable/integrations/pam_and_nsswitch.html
